zerberus/zerberus.cpp

```diff
@@ -447,6 +447,11 @@ bool Zerberus::loadInstrument(const QString& s)
                   busy = false;
                   return true;
                   }
+            else
+                  qDebug("instr->load(path) returned false");
```

Comment:
I acknowledge this is maybe a superfluous qDebug message that I maybe I should remove...but I spent a lot of time debugging what happens when I load more soundfonts than can allocate memory for and needed to know why the loadInstrument function failed.  I think should leave this qDebug message, because the popup dialog does not provide sufficient information to end user about why the load failed.